pkg/services/folder/folderimpl/sqlstore.go

```diff
@@ -24,30 +24,59 @@ func ProvideStore(db db.DB, cfg *setting.Cfg, features featuremgmt.FeatureManage
 	return &sqlStore{db: db, log: log.New("folder-store"), cfg: cfg, fm: features}
 }
 
-func (s *sqlStore) Create(ctx context.Context, cmd *folder.CreateFolderCommand) (*folder.Folder, error) {
+func (ss *sqlStore) Create(ctx context.Context, cmd *folder.CreateFolderCommand) (*folder.Folder, error) {
 	panic("not implemented")
 }
 
-func (s *sqlStore) Delete(ctx context.Context, uid string, orgID int64) error {
-	panic("not implemented")
+func (ss *sqlStore) Delete(ctx context.Context, uid string, orgID int64) error {
+	return ss.db.WithDbSession(ctx, func(sess *db.Session) error {
+		_, err := sess.Exec("DELETE FROM dashboard WHERE folder_uid=? AND org_id=?", uid, orgID)
```

Comment:
This needs to delete the record from the new folder table, not the dashboard table :)